samples/csharp_dotnetcore/50.teams-messaging-extensions-search/Bots/TeamsMessagingExtensionsSearchBot.cs

```diff
@@ -99,5 +136,153 @@ private async Task<IEnumerable<(string, string, string, string, string)>> FindPa
             var obj = JObject.Parse(await (new HttpClient()).GetStringAsync($"https://azuresearch-usnc.nuget.org/query?q=id:{text}&prerelease=true"));
             return obj["data"].Select(item => (item["id"].ToString(), item["version"].ToString(), item["description"].ToString(), item["projectUrl"]?.ToString(), item["iconUrl"]?.ToString()));
         }
+
+
+        public MessagingExtensionResponse GetAdaptiveCard()
+        {
+
+            string filepath = "./RestaurantCard.json";
+
+            //string filepath = "./WeatherCard.json";
+
+            var previewcard = new ThumbnailCard
+            {
+                Title = "Adaptive Card",
+                Text = "Please select to get Adaptive card"
+            };
+
+            var adaptiveList = FetchAdaptive(filepath);
+            var attachment = new MessagingExtensionAttachment
+            {
+                ContentType = AdaptiveCards.AdaptiveCard.ContentType,
+                Content = adaptiveList.Content,
+                Preview = previewcard.ToAttachment()
+            };
+
+            // return attachment;
+
+            return new MessagingExtensionResponse
+            {
+                ComposeExtension = new MessagingExtensionResult
+                {
+                    Type = "result",
+                    AttachmentLayout = "list",
+                    Attachments = new List<MessagingExtensionAttachment> { attachment }
+                }
+            };
+        }
+
+
+        public MessagingExtensionResponse GetConnectorCard()
+        {
+          
+            string filepath = "./connectorCard.json";
```

Comment:
I think this will not work as expected on a mac or linux machine.

In the CardsBot sample, resources are loaded using this cross platform code:

```cs
            // combine path for cross platform support
            var paths = new[] { ".", "Resources", "adaptiveCard.json" };
            var adaptiveCardJson = File.ReadAllText(Path.Combine(paths));

            var adaptiveCardAttachment = new Attachment()
            {
                ContentType = "application/vnd.microsoft.card.adaptive",
                Content = JsonConvert.DeserializeObject(adaptiveCardJson),
            };
```

CoreBot uses this method (which is also cross platform):

```cs
        // Load attachment from embedded resource.
        private Attachment CreateAdaptiveCardAttachment()
        {
            var cardResourcePath = "CoreBot.Cards.welcomeCard.json";

            using (var stream = GetType().Assembly.GetManifestResourceStream(cardResourcePath))
            {
                using (var reader = new StreamReader(stream))
                {
                    var adaptiveCard = reader.ReadToEnd();
                    return new Attachment()
                    {
                        ContentType = "application/vnd.microsoft.card.adaptive",
                        Content = JsonConvert.DeserializeObject(adaptiveCard),
                    };
                }
            }
        }
```